.travis.yml

```diff
@@ -21,9 +21,9 @@ matrix:
           addons: { apt: { packages: ["valgrind", "gcc-5", "g++-5"], sources: ["ubuntu-toolchain-r-test"] } }
           env: MY_CC=gcc-5 MY_CXX=g++-5
 
-        # Test our minium Clang version, which is clang 3.4 on Ubuntu Trusty
+        # Test our minium Clang version, which is clang 3.5 on Ubuntu Trusty
         - os: linux
-          addons: { apt: { packages: ["valgrind", "clang-3.4", "libc++-dev", "libc++abi-dev"], sources: ["ubuntu-toolchain-r-test"] } }
+          addons: { apt: { packages: ["valgrind", "clang-3.5", "libc++-dev", "libc++abi-dev", "g++-4.9"], sources: ["ubuntu-toolchain-r-test"] } }
```

Comment:
The line you’re concerned with in .travis.yml is because of this:

/usr/bin/../lib/gcc/x86_64-linux-gnu/4.8/../../../../include/c++/4.8/cstdio:120:11: error: no member named 'gets' in the global namespace
  using ::gets;
It took a combination of updating the minimum version of Clang to 3.5 and the addition of the g++-4.9 to get rid of that. If I only updated Clang, I got this error:


/usr/bin/../lib/gcc/x86_64-linux-gnu/4.9/../../../../include/c++/4.9/cstddef:51:11: error: no member named 'max_align_t' in the global namespace
  using ::max_align_t;
Further discussions about those errors can be found here:

https://stackoverflow.com/questions/17775390/clang-3-3-in-c1y-mode-cannot-parse-cstdio-header
catchorg/Catch2#334
Those changes were only required for the minimum requirements for Ubuntu Trusty. I discussed these changes to the minimum versions with Abhinay and he didn’t think it would be much of a problem. I’m sorry if I should’ve discussed it with you first.